site-src/geps/gep-1651.md

```diff
@@ -0,0 +1,40 @@
+# GEP-1651: Cluster Local Gateways
+
+* Issue: [#1651](https://github.com/kubernetes-sigs/gateway-api/issues/1651)
+* Status: Provisional
+
+(See status definitions [here](overview.md#status).)
+
+## TLDR
+
+Allow users to configure a Gateway and it's attached Routes to not
+be exposed externally.
+
+## Goals
+
+- Allow Gateway and Routes to be configured directly to not be exposed externally
```

Comment:
"Not be exposed externally" sounds slightly less specific than "Cluster local". For example, Cloud internal load balancers are not exposed "externally", but are often accessible from outside of the cluster as long as you're in the same VPC. 

In the Cloud ILB case, the best solution has seemed to be to create a separate GatewayClass to represent that. Would the same concept apply here?

If I understand the original discussion correctly, I think the primary motivation was attaching a ClusterIP to an in-cluster Gateway, and not a LB. If that's the case, would MultiCluster Services (SuperClusterIP) fit?

Reply:
Or just another Gateway with the `address` set to an internal thing

Reply:
> "Not be exposed externally" sounds slightly less specific than "Cluster local".
I'll update the goal to mention the access is cluster local.

> In the Cloud ILB case, the best solution has seemed to be to create a separate GatewayClass to represent that. Would the same concept apply here?

Pushing this into the GatewayClass means it becomes implementation specific and doesn't really help end-users. 

> If I understand the original discussion correctly, I think the primary motivation was attaching a ClusterIP to an in-cluster Gateway, and not a LB. 

Yup

> If that's the case, would MultiCluster Services (SuperClusterIP) fit?

Are you referring to [KEP-1645: Multi-Cluster Services API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)? 

If so then no - the goals in the KEP seem to be about exposing services across clusters. The scope of this GEP is a single cluster.

Reply:
Updated the language to emphasis `cluster local`'s scope is within a single cluster.